### PR TITLE
Create these-de-medecine-paris-xi.csl

### PR DESCRIPTION
**Pull request by Jishin42**
New style added for French Medecine Thesis, based on http://www.zotero.org/styles/medecine-sciences.

### CODE REVIEW

File: these-de-medecine-paris-xi.csl [hunk 1]
```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" default-locale="fr-FR">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>These de medecine Paris XI</title>
+    <id>http://www.zotero.org/styles/these-de-medecine-paris-xi</id>
+    <link href="http://www.zotero.org/styles/these-de-medecine-paris-xi" rel="self"/>
+    <link href="http://www.zotero.org/styles/medecine-sciences" rel="template"/>
+    <link href="http://www.zotero.org/styles/plos" rel="template"/>
+    <link href="http://www.medecinesciences.org" rel="documentation"/>
```
Comment (adam3smith) [inline these-de-medecine-paris-xi.csl]: is the style for Paris XI documented anywhere? We really would like online documentation, especially for styles that don't correspond to any easily accessible publications like journal styles.

File: these-de-medecine-paris-xi.csl [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    <names variable="editor" delimiter=", ">
+      <name initialize-with="" name-as-sort-order="all" sort-separator=" " delimiter=", " delimiter-precedes-last="always"/>
+      <label form="long" prefix=", "/>
+    </names>
+  </macro>
+  <macro name="author">
+    <names variable="author" suffix=".">
+      <name name-as-sort-order="all" sort-separator=" " initialize-with="" delimiter=", " delimiter-precedes-last="always"/>
+      <label form="short" prefix=" "/>
+      <et-al />
```
Comment (adam3smith) [inline these-de-medecine-paris-xi.csl]: delete this line -- without attribute, et-al doesn't do anything.

File: these-de-medecine-paris-xi.csl [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+		<else>
+          <text macro="title" prefix=" " suffix="."/>
+          <text variable="container-title" form="short"  strip-periods="true" prefix=" " suffix="."/>
+          <text macro="year-date" prefix="" suffix=";"/>
+          <choose>
+		    <if match="none" variable="volume">
+              <text variable="DOI" prefix="doi:"/>
+            </if>
+            <else>
+			  <text variable="volume" prefix="" suffix=""/>
```
Comment (adam3smith) [inline these-de-medecine-paris-xi.csl]: remove empty prefix/suffix

**Comment by adam3smith** [comment]
Otherwise this looks fine. My my concern is actually the documentation.

File: these-de-medecine-paris-xi.csl (hunk 1 above)
Comment (rmzelle) [inline these-de-medecine-paris-xi.csl]: It also looks like Paris XI is better known as "Université Paris-Sud".

Comment (Jishin42) [inline these-de-medecine-paris-xi.csl]: The only documentation I have is a document given by the university. It is in French.
I uploaded it on a personal cloud (http://ovh.to/hCBLzv6) but is there a common repository for this kind of document ? Should I commit it in this repository ?

Comment (adam3smith) [inline these-de-medecine-paris-xi.csl]: as long as you can keep the link to the document on the cloud permanent, that's great. We'd rather not host style manuals here, no.

**Comment by rmzelle** [comment]
We need to come up with a better name for this style, as well. Do I understand correctly that this citation style's scope is limited to the Diplôme d'Etudes Spécialisées (DES) de Psychiatrie of the Université Paris-Sud?

**Comment by rmzelle** [comment]
We're still willing to accept this once we figure out a good name for the style.

Closing now because of inactivity (if we hear back from you we can reopen).

**Pull request closed without merging**